src/Library/demos/Drop Zone/main.blp

```diff
@@ -0,0 +1,33 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: _("Drop Zone");
+  description: _("A versatile drop area for external content");
+
+  Box {
+    orientation: vertical;
+
+    Adw.Clamp {
+      maximum-size: 500;
+
+      Adw.Bin bin {
```

Comment:
`Bin` isn't really meant to be used directly but rather for subclassing

Would using `Box` complicate the code significantly?

Reply:
It's much easier to use a bin as I can just set the child parameter, with a box i would have to handle the removing a child from the box whenever another item is dropped. The document page said that "it is useful for deriving subclasses" does that mean we have to?

Reply:
Let's use it then :+1: